website/public/js/services/userServices.js

```diff
@@ -201,6 +198,16 @@ angular.module('habitrpg')
           })
         },
 
+        clearNewMessages: function () {
+          $http({
+            method: "POST",
+            url: 'api/v3/user/mark-pms-read',
+          })
+          .then(function (response) {
+            sync();
```

Comment:
I think in this case a common op is worth it, in the case of debug options that'll be used only by devs syncing is okay